stablehlo/dialect/Base.h

```diff
@@ -56,6 +56,8 @@ inline static bool isStaticDimSize(int64_t val) {
   return !isDynamicDimSize(val);
 }
 
+SmallVector<int64_t> i64ArrayOrElementsValues(Attribute);
```

Comment:
Let's phrase this as a getter function:

```
// Returns an array ref from 1-D DenseIntElementsAttrs or DenseI64ArrayAttr
ArrayRef<int64_t> getI64Array(Attribute);
```

Also can we use this getter function in the generated code? As long as it is defined before the `inc` is included it should be viable, I believe everything in `Base.h` is already included and can be used in generated code.